src/pages/index.js

```diff
@@ -135,42 +261,57 @@ function handleImageClick({ name, link }) {
   popupWithImage.open({ name, link });
 }
 
-function getCardElement(cardData) {
-  const cardElement = cardTemplate.cloneNode(true);
-  const cardImageEl = cardElement.querySelector(".card__image");
-  const cardTitleEl = cardElement.querySelector(".card__title");
-  const likeButton = cardElement.querySelector(".card__like-button");
-  const deleteButton = cardElement.querySelector(".card__delete-button");
-
-  // Set up card content
-  cardImageEl.src = cardData.link;
-  cardImageEl.alt = cardData.name;
-  cardTitleEl.textContent = cardData.name;
-
-  // Add event listeners
-  likeButton.addEventListener("click", () => {
-    likeButton.classList.toggle("card__like-button_active");
-  });
-  // choose between redundant code function on 137
-  cardImageEl.addEventListener("click", () => {
-    console.log("heres your card click event bro");
-    modalImage.src = cardImageEl.src;
-    modalImage.alt = cardData.name;
-    modalCaption.textContent = cardData.name;
-    imgModal.open();
-  });
-
-  //deleteButton.addEventListener("click", handleDeleteCard);
-
-  function handleDeleteCard() {
-    cardElement.remove();
-  }
-  return cardElement;
+function handleLikeClick(card, cardId) {
+  api
+    .likeCard(cardId)
+    .then(() => {
+      card.updateLikeButton();
+    })
+    .catch((err) => console.error(`Error adding card: ${err}`));
```

Comment:
Here you **put** the like, but we should be able to **dislike** a card if the card is liked